src/Gzero/Core/CMSSeeder.php

```diff
@@ -214,15 +214,19 @@ private function seedUsers()
         // Create user
         $user = User::firstOrCreate(
             [
-                'email'     => 'a@a.pl',
+                'email'     => 'johndoe@gzero.com',
```

Comment:
Why did you changed this email? If we want to use some real domain, we could use gzero.pl (maybe admin@gzero.pl?)